src/rpc/blockchain.cpp

```diff
@@ -1924,6 +1924,37 @@ static UniValue savemempool(const JSONRPCRequest& request)
     return NullUniValue;
 }
 
+static UniValue clearmempool(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 0) {
+        throw std::runtime_error(
+            "clearmempool\n"
+            "\nRemoves all transactions from the mempool.\n"
+            "It will fail if a previous dump is still being loaded.\n"
+            "\nResult:\n"
+            "{                           (json object)\n"
+            "  \"size\": n,                (numeric) The number of evicted transactions\n"
+            "}\n"
+            "\nExamples:\n"
+            + HelpExampleCli("clearmempool", "")
+            + HelpExampleRpc("clearmempool", "")
+        );
+    }
+
+    if (!g_is_mempool_loaded) {
+        throw JSONRPCError(RPC_MISC_ERROR, "The mempool was not loaded yet");
+    }
+
+    LOCK(mempool.cs);
+
+    UniValue result(UniValue::VOBJ);
+    result.pushKV("size", mempool.size());
```

Comment:
maybe call this `transactions_removed` or something more informative

Reply:
Changed.  `size` is consistent with `getmempoolinfo`, but in that context "size" is clearly more descriptive than for `clearmempool`.  I agree that here `transactions_removed` is a better name.